docs/test-plan.md

```diff
@@ -152,6 +152,22 @@ Run one of the above MRVAs, but cancel it from within VS Code:
    - Check that the editor loads and shows methods to model.
    - Check that methods are grouped per library (e.g. `rocksdbjni@7.7.3` or `asm@6.0`)
    - Check that the "Open database" link works.
+   - Check that a method can have multiple models:
+      - add a couple of new models for one method
+      - save and check that the modeling file (use the 'open extension pack' button to open it) shows multiple methods
+      - check that a 'duplicated classification' error appears in both model editor and modeling panel when a duplicate modeling occurs
+      - check that a 'conflicting classigication' error appears when a neutral model type is paired with a source kind
```

Comment:
Neutral models aren't conflicting with source kinds only, they are conflicting with models that have the same `type` as the neutral model's `kind`. What about:

```suggestion
      - check that a 'conflicting classification' error appears when a neutral model type is paired with a model of the same kind
```